core/assets/bundles/bundle_id_ID.properties

```diff
@@ -2120,7 +2120,7 @@ onset.fabricator = Gunakan [accent]unit[] untuk menjelajah peta, mempertahakan b
 onset.makeunit = Produksi sebuah unit.\nGunakan tombol "?" untuk melihat persyaratan pabrik yang dipilih.
 onset.turrets = Unit sangatlah efektif, namun [accent]menara[] memberikan kemampuan pertahanan yang lebih baik jika digunakan secara efektif.\nTempatkan menara :breach: [accent]Breach[].\nMenara memerlukan amunisi :beryllium: [accent]berilium[].
 onset.turretammo = Suplai menara dengan [accent]amunisi berilium.[]
-onset.walls = [accent]Dinding[] dapat mencegah kerusakan yang datang pada bangunan.\nTempatkan beberapa :beryllium-wall: [accent]dinding berilium[] di sekitar menara.
+onset.walls = [accent]Dinding[] dapat mencegah damage yang datang pada bangunan lain.\nTempatkan beberapa :beryllium-wall: [accent]dinding berilium[] di sekitar menara.
 onset.enemies = Musuh datang, bersiaplah untuk bertahan.
```

Comment:
Replace dot with exclamation mark, it was imperative and also the situation was urgent, same as the gz.wave3 review

Reply:
and also the comma though

Reply:
The line kept unchanged.